cmd/inspector/main.go

```diff
@@ -79,15 +80,36 @@ var (
 		Args:  cobra.MinimumNArgs(4),
 		RunE:  SegmentHealth,
 	}
+	paymentsCmd = &cobra.Command{
+		Use:   "payments",
+		Short: "commands for payments",
+	}
+	prepareInvoiceRecordsCmd = &cobra.Command{
+		Use:   "prepare-invoice-records <period>",
+		Short: "Prepares invoice project records that will be used during invoice line items creation",
+		Args:  cobra.MinimumNArgs(1),
+		RunE:  prepareInvoiceRecords,
+	}
+	createInvoiceItemsCmd = &cobra.Command{
+		Use:   "create-invoice-items",
+		Short: "Creates stripe invoice line items for not consumed project records",
+		RunE:  createInvoiceItems,
+	}
+	createInvoicesCmd = &cobra.Command{
+		Use:   "create-invoices",
+		Short: "Creates stripe invoices for all stripe customers known to satellite",
+		RunE:  createInvoices,
+	}
 )
 
 // Inspector gives access to overlay.
 type Inspector struct {
-	conn          *rpc.Conn
-	identity      *identity.FullIdentity
-	overlayclient rpc.OverlayInspectorClient
-	irrdbclient   rpc.IrreparableInspectorClient
-	healthclient  rpc.HealthInspectorClient
+	conn           *rpc.Conn
+	identity       *identity.FullIdentity
+	overlayclient  rpc.OverlayInspectorClient
+	irrdbclient    rpc.IrreparableInspectorClient
+	healthclient   rpc.HealthInspectorClient
+	paymentsClient rpc.PaymentsClient
```

Comment:
Why would this be part of an inspector? Inspector is all about inspecting not changing the state.

Reply:
Once @jtolds  told me that we should place it there

Reply:
When I use inspector I would expect that when I call anything on it, then it's safe to do concurrently.